docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-immutable-con.adoc

```diff
@@ -0,0 +1,15 @@
+[id='environment-immutable-con']
+= Environment with immutable servers
+You can deploy an environment that includes several different pods running {KIE_SERVER} with preloaded processes. The database servers are, by default, also run in pods. Each {KIE_SERVER} pod can be separately scaled as necessary.
+
+In this case, any processes (KJAR files) must be loaded onto a {KIE_SERVER} at the time the image is created. You can not load or unload processes on a running {KIE_SERVER}. The advantage of this approach is that the {KIE_SERVER} with the processes in it runs like any other containerized service and does not require specialized management. You can use existing approaches to cloud integration to deploy and manage the servers.
+
+Optionally, you can also deploy a pod with {CENTRAL} Monitoring and pod with Smart Router are also deployed. You can optionally use {CENTRAL} Monitoring to start and stop (but not deploy) processes on your {KIE_SERVERS} and to view monitoring data. 
```

Comment:
- Optionally, you can also deploy a pod with {CENTRAL} Monitoring and pod with Smart Router are also deployed -> Optionally, you can also deploy pods with {CENTRAL} Monitoring and Smart Router.
- You can optionally use Business Central Monitoring -> You can use Business Central Monitoring